Assets/Scripts/NursePatrol.cs

```diff
@@ -60,9 +63,22 @@ void GotoNextPoint()
     // Update is called once per frame
     void Update()
     {
+
+        float distance = Vector3.Distance(transform.position, spy.transform.position);
+        // Chase spy
+        if (distance < agentChaseDistance && chase)
+        {
+            Vector3 dirToSpy = transform.position - spy.transform.position;
+            Vector3 newPos = transform.position - dirToSpy;
+
+            agent.SetDestination(newPos);
+        }
```

Comment:
So to be clear, this does **not** use Chadley's "field of vision" detection code?
(See https://github.com/FinnbarrOC/EspionAge/blob/345d486d3c30bad6d3686c65e146eb606c8d29d3/Assets/Scripts/FieldOfVision.cs#L61)

If so, I can reject this PR and quickly add chasing using Chadley's AI vision system.

Reply:
I'm making it with chadley's vision system now! So these will be overwritten in a little bit!

Reply:
We had some more discussion on discord and I didn't realize that we were using that, we're talking in #dev channel!! :)